calculation.js

```diff
@@ -0,0 +1,10 @@
+function strictEqual(val1, val2) {
+    //returns true when the parameters are of the same value and type  
+    if ((val1 == val2) && (typeof val1 == typeof val2 )) {
+        return true
+    }
+    
+    else return false
+}
```

Comment:
Good solution. Notice though that we are saying if the condition is true return true and if the condition is false return false. This means we can just return the condition.

```JavaScript
function strictEqual(val1, val2) {
    return val1 == val2 && typeof val1 == typeof val2
}
```

With that now being a one line with only a return we could use ES6+ syntax to take advantage of fat arrow's implicit return

```JavaScript
const strictEqual = (val1, val2) => val1 == val2 && typeof val1 == typeof val2
```